brightlayer-ui/icons
# Remove dependency on @mui/styles

Fixes #232. 


#### Changes proposed in this Pull Request:
- Remove dependency on @mui/styles


#### To Test:
- cd mui && yarn start
- Make sure the demo still works and all of the icons are rendered
